workspaces/ui/src/entrypoints/testing-sessions.js

```diff
@@ -17,10 +26,71 @@ export default function TestingSessions(props) {
     exampleSessionCollection: 'example-sessions',
   });
 
+  const captureServiceFactory = async (specService, captureId) => {
+    return new ExampleCaptureService(specService);
+  };
+
+  const diffServiceFactory = async (
+    specService,
+    captureService,
+    _rfcState,
+    additionalCommands,
+    config,
+    captureId
+  ) => {
+    async function computeInitialDiff() {
```

Comment:
@devdoshi this is the last thing blocking example sessions. Line 43 always gets events '[]' even after you make changes. Can you look at lifecycle here?